### PR TITLE
Update CSI user config workflow 

### PR DESCRIPTION
**Pull request by copejon**
This PR aligns the user workflow described here with the actual state of the project./

### CODE REVIEW

File: enhancements/microshift/microshift-default-csi-plugin.md [hunk 1]
```diff
@@ -143,10 +143,7 @@ The workflow is automated and documented
 
 > After the device owner has logged into the device ... 
 
-1. The device owner will create a file called `lvmd.yaml`, per documentation according to its 
-[documentation](https://github.com/red-hat-storage/topolvm/blob/main/docs/lvmd.md).
-   1. The device owner will provide the absolute path to this file in the MicroShift config field `LvmdConfigFilePath`. 
-
+1. The device owner will create a file called `lvmd.yaml` _in the same directory as the https://github.com/openshift/microshift/blob/main/docs/howto_config.md_, specified per the lvmd [documentation](https://github.com/red-hat-storage/topolvm/blob/main/docs/lvmd.md).
```
Comment (dhellmann) [inline enhancements/microshift/microshift-default-csi-plugin.md:146]: It looks like the URL is in there twice, is that intentional? Or should that read "in the same directory as the config file"?

Comment (copejon) [reply enhancements/microshift/microshift-default-csi-plugin.md:146]: Ah, will fix

File: enhancements/microshift/microshift-default-csi-plugin.md [hunk 2]
```diff
@@ -143,10 +143,7 @@ The workflow is automated and documented
 
 > After the device owner has logged into the device ... 
 
-1. The device owner will create a file called `lvmd.yaml`, per documentation according to its 
-[documentation](https://github.com/red-hat-storage/topolvm/blob/main/docs/lvmd.md).
-   1. The device owner will provide the absolute path to this file in the MicroShift config field `LvmdConfigFilePath`. 
-
+1. The device owner will create a file called `lvmd.yaml` in the same directory as the [microshift config](https://github.com/openshift/microshift/blob/main/docs/howto_config.md), specified per the lvmd [documentation](https://github.com/red-hat-storage/topolvm/blob/main/docs/lvmd.md).
```
Comment (ggiguash) [inline enhancements/microshift/microshift-default-csi-plugin.md:146]:
```suggestion
1. The device owner will create a file called `lvmd.yaml` in the same directory as the [MicroShift config](https://github.com/openshift/microshift/blob/main/docs/howto_config.md), specified per lvmd [documentation](https://github.com/red-hat-storage/topolvm/blob/main/docs/lvmd.md).
```

**Pull request merged**